library/std/src/sys/stdio/solid.rs

```diff
@@ -1,22 +1,13 @@
+#[allow(dead_code)]
```

Comment:
If you know that there is dead code, `expect` is better since detects if the attribute becomes redundant.
```suggestion
#[expect(dead_code)]
```